advanced/Scripts/updatecheck.sh

```diff
@@ -35,8 +49,10 @@ rm -f "/etc/pihole/localversions"
 
 # Create new versions file if it does not exist
 VERSION_FILE="/etc/pihole/versions"
-touch "${VERSION_FILE}"
-chmod 644 "${VERSION_FILE}"
+if [ ! -f ${VERSION_FILE} ]; then
```

Comment:
No need to guard this, see https://github.com/pi-hole/pi-hole/pull/4875#discussion_r951935177